Makefile

```diff
@@ -70,6 +70,7 @@ check-all-committed: ## Fail in case there are uncommitted changes
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/manager/main.go
+	go build -o bin/csi-addons ./cmd/csi-addons
```

Comment:
do we need to place this in `tools`/`debug` dir?

Reply:
Some documentation is included in the directory where the tool has its files? Documentation for executables can probably best be placed alongside their `main.go` file.